manifests/40-rbac.yaml

```diff
@@ -66,7 +66,7 @@ rules:
   verbs: ["create","get","list","watch"]
 # Needed by every CVO-managed operator.
```

Comment:
This comment is no longer true based on the "infrastructures/status" addition, can we adjust it?

Reply:
That's a good point, Since I think we want to revisit this permission, I moved it to it's own stanza and added a TODO for it.